src/lib.rs

```diff
@@ -124,6 +124,7 @@ pub mod prelude {
     pub use crate::secret::{Password, PrivateKey};
     pub use crate::transaction::{Account, GasPrice};
     pub use web3::api::Web3;
+    #[cfg(feature = "http")]
     pub use web3::transports::Http;
```

Comment:
Should we remove this re-export alltogether?

Reply:
Hmm, I think this is the most commonly used transport, so its nice to have it in the `prelude`. But I'm also don't have a strong opinion.